BuberBreakfast/BuberBreakfast.csproj

```diff
@@ -4,6 +4,10 @@
     <ProjectReference Include="..\BuberBreakfast.Contracts\BuberBreakfast.Contracts.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
# painful lesson
All of these dependencies are required are in here. Don't bother installing sub-packages like `MySql.EntityFrameworkCore.Tools`